src/progressive-profiling/OrganizationFormField.jsx

```diff
@@ -0,0 +1,168 @@
+import React, {
+  useCallback,
+  useEffect, useMemo,
+  useRef,
+} from 'react';
+
+import { useIntl } from '@edx/frontend-platform/i18n';
+import { Form } from '@edx/paragon';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { debounce } from 'lodash';
```

Comment:
just curios what does it used for

Reply:
Otherwise, the linter constantly emphasizes to me that I supposedly do not have this module installed, although it is installed and works great.
`'lodash' should be listed in the project's dependencies. Run 'npm i -S lodash' to add iteslint`

Reply:
I removed this line